source/renderer/app/stores/WalletsStore.js

```diff
@@ -918,6 +950,14 @@ export default class WalletsStore extends Store {
           this.goToWalletRoute(this.active.id);
         }
       }
+      if (this.active && WALLET_PUBLIC_KEY_SHARING_ENABLED) {
```

Comment:
@yakovkaravelov I would prefer if we would load the wallet public key either when:
A) User navigates to wallet "Settings" tab, or even better
B) User clicks on the "Reveal" button

Reply:
Fixed.